src/analysis/imports_sortedness.d

```diff
@@ -66,7 +66,7 @@ class ImportSortednessCheck : BaseAnalyzer
 
 private:
 
-	int level = 0;
```

Comment:
Speaking of that i still have a PR opened that make the check very reliable. (#475)